scripts/common/sonic-telemetry-build/build.sh

```diff
@@ -15,14 +13,11 @@ sudo service redis-server start
 # Install libyang
 sudo dpkg -i buildimage/target/debs/stretch/libyang*.deb
 
-# Clone sonic-mgmt-common repository
-git clone https://github.com/Azure/sonic-mgmt-common.git
-
 # Build sonic-mgmt-common first
 
 pushd sonic-mgmt-common
 
-dpkg-buildpackage -rfakeroot -b -us -uc
+NO_TEST_BINS=1 dpkg-buildpackage -rfakeroot -b -us -uc
```

Comment:
can you remind me what this notestbin is?

Reply:
It used by sonic-mgmt-common's makefiles to determine whether to build go test cases o not.
Telemetry build will not run sonic-mgmt-common's test cases and hence it need not compile them.